include/emds/emds.h

```diff
@@ -163,12 +163,19 @@ uint32_t emds_store_time_get(void);
  */
 uint32_t emds_store_size_get(void);
 
+/**
+ * @brief Check if the store operation can be run.
+ *
+ * @return Whether the store operation can be stared or not.
```

Comment:
```suggestion
 * @return True if the store operation can be started, otherwise false.
```
Nitpick/matter of opinion. If you decide to change it there is a simmilar doc entry for `emds_is_storing`